static/src/javascripts/projects/common/modules/commercial/ticker.js

```diff
@@ -108,69 +145,91 @@ const populateGoal = (parentElement: HTMLElement, tickerType: TickerType) => {
+            countElement.innerHTML = `${getCurrencySymbol(tickerSettings)}${amount.toLocaleString()}`;
 
             if (goalReached()) {
-                labelElement.innerHTML = 'contributed';
+                labelElement.innerHTML = tickerSettings.copy.countLabel;
             }
         }
     }
 };
 
-const animate = (parentElementSelector: string, tickerType: TickerType) => {
+const animate = (parentElementSelector: string, tickerSettings: TickerSettings) => {
     const parentElement = document.querySelector(parentElementSelector);
 
     if (parentElement && parentElement instanceof HTMLElement) {
         if (goalReached()) {
             parentElement.classList.add('epic-ticker__goal-reached');
         }
 
-        populateGoal(parentElement, tickerType);
+        populateGoal(parentElement, tickerSettings);
 
         window.setTimeout(() => {
             count[parentElementSelector] = 0;
             window.requestAnimationFrame(() =>
                 populateStatusSoFar(
                     parentElementSelector,
                     parentElement,
-                    tickerType
+                    tickerSettings,
                 )
             );
-            animateBar(parentElement, tickerType);
+            animateBar(parentElement, tickerSettings.endType);
         }, 500);
 
-        parentElement.classList.add(`epic-ticker__${tickerType}`);
+        parentElement.classList.add(`epic-ticker__${tickerSettings.endType}`);
         parentElement.classList.remove('is-hidden');
     }
 };
 
 const dataSuccessfullyFetched = () =>
     !(Number.isNaN(Number(total)) || Number.isNaN(Number(goal)));
 
+const getTickerUrl = (countType: TickerCountType): string =>
+    countType === 'people' ? 'https://support.theguardian.com/supporters-ticker.json' : 'https://support.theguardian.com/ticker.json';
+
 const fetchDataAndAnimate = (
     parentElementSelector: string,
-    tickerType: TickerType
+    tickerSettings: TickerSettings,
 ) => {
     if (dataSuccessfullyFetched()) {
-        animate(parentElementSelector, tickerType);
+        animate(parentElementSelector, tickerSettings);
     } else {
-        fetchJSON('https://support.theguardian.com/ticker.json', {
+        fetchJSON(getTickerUrl(tickerSettings.countType), {
             mode: 'cors',
         }).then(data => {
             total = parseInt(data.total, 10);
             goal = parseInt(data.goal, 10);
 
             if (dataSuccessfullyFetched()) {
-                animate(parentElementSelector, tickerType);
+                animate(parentElementSelector, tickerSettings);
             }
         });
     }
 };
 
-export const initTicker = (
+const defaultSettings: TickerSettings = {
+    endType: 'unlimited',
+    countType: 'people',
+    copy: {
+        countLabel: 'supporters in Australia',
+        goalReachedPrimary: 'We\'ve hit our goal!',
+        goalReachedSecondary: 'but you can still support us',
+    },
+    currencySymbol: '$',
```

Comment:
Does it make sense to have a default `currencySymbol` here if the `countType` is people? I think it won't be used either way, but _could_ be confusing?

Reply:
You're right!
I started out with this as optional, but decided it's simpler to make it mandatory than do extra validation